vi/news/_posts/2016-11-15-ruby-2-2-6-released.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: news_post
+title: "Ruby ra mắt ấn bản 2.2.6"
+author: "usa"
+translator: "Trung Lê"
+date: 2016-11-15 12:00:00 +0000
+lang: vi
+---
+
+Ruby 2.2.6 mới được phát hành
+
+Bản này kèm SSL certificates mới cho RubyGems và 80 vá lỗi.
```

Comment:
It should be:

"Bản này có kèm theo SSL certificates mới cho RubyGems, và fix khoảng 80 lỗi khác từ bản release trước."